pkg/services/ngalert/metrics/scheduler.go

```diff
@@ -38,47 +34,15 @@ func NewSchedulerMetrics(r prometheus.Registerer) *Scheduler {
 			Name:      "scheduler_behind_seconds",
 			Help:      "The total number of seconds the scheduler is behind.",
 		}),
-		// TODO: once rule groups support multiple rules, consider partitioning
-		// on rule group as well as tenant, similar to loki|cortex.
-		EvalTotal: promauto.With(r).NewCounterVec(
-			prometheus.CounterOpts{
-				Namespace: Namespace,
-				Subsystem: Subsystem,
-				Name:      "rule_evaluations_total",
-				Help:      "The total number of rule evaluations.",
-			},
-			[]string{"org"},
-		),
-		// TODO: once rule groups support multiple rules, consider partitioning
-		// on rule group as well as tenant, similar to loki|cortex.
-		EvalFailures: promauto.With(r).NewCounterVec(
-			prometheus.CounterOpts{
-				Namespace: Namespace,
-				Subsystem: Subsystem,
-				Name:      "rule_evaluation_failures_total",
-				Help:      "The total number of rule evaluation failures.",
-			},
-			[]string{"org"},
-		),
-		EvalDuration: promauto.With(r).NewHistogramVec(
-			prometheus.HistogramOpts{
-				Namespace: Namespace,
-				Subsystem: Subsystem,
-				Name:      "rule_evaluation_duration_seconds",
-				Help:      "The time to evaluate a rule.",
-				Buckets:   []float64{.01, .1, .5, 1, 5, 10, 15, 30, 60, 120, 180, 240, 300},
-			},
-			[]string{"org"},
-		),
-		ProcessDuration: promauto.With(r).NewHistogramVec(
-			prometheus.HistogramOpts{
+		// TODO: partition on rule group as well as tenant, similar to loki|cortex.
+		GroupRules: promauto.With(r).NewGaugeVec(
+			prometheus.GaugeOpts{
 				Namespace: Namespace,
 				Subsystem: Subsystem,
-				Name:      "rule_process_evaluation_duration_seconds",
-				Help:      "The time to process the evaluation results for a rule.",
-				Buckets:   []float64{.01, .1, .5, 1, 5, 10, 15, 30, 60, 120, 180, 240, 300},
+				Name:      "rule_group_rules",
+				Help:      "The number of alert rules that are scheduled, both active and paused.",
 			},
-			[]string{"org"},
+			[]string{"org", "state"},
```

Comment:
It looks like I've added code here, but it's just because these lines got moved in the file.